src/interaction/interactiveTarget.js

```diff
@@ -42,5 +49,11 @@ module.exports = {
      * @todo Needs docs.
      * @private
      */
-    _touchDown: false
+    _touchDown: false,
+    
+    /**
+     * Used to track the global start position of a touchdown event on this target.
+     * @private
+     */
+    _touchDownPoint: new core.Point()
```

Comment:
Actually answering my own question, since this is not a great pattern to promote. I would suggest making `touchDownPoint` a privately scoped static object within InteractionManager.js, and use that instead. No need to have it accessible on all display objects.